lib/ansible/modules/cloud/openstack/os_client_config.py

```diff
@@ -28,7 +28,7 @@
           information on all configured clouds
      required: false
      default: []
-requirements: [ os-client-config ]
+requirements: [ openstacksdk ]
```

Comment:
Would people have `openstacksdk` already if they have `os-client-config`?
Wonder if this should go into the porting guide

https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.8.html

Wonder if this means it's a feature PR, which is for `devel` and not for backporting?

Reply:
A quick hunt around shows that Debian makes python-os-client-config depend on python-openstacksdk (in Sid), however Jessie/Stretch do not require openstacksdk. Similarly, Ubuntu Xenial/Bionic do not require it. Installing os-client-config from pip does pull in openstacksdk (under python3 anyway).

Reply:
Thinking about it more, I think it could be problematic to introduce this as a backport as it's not a given that the user will have openstacksdk installed. That being said, the module does return a sensible message back to the user if it's not present and will fall back to os-client-config if that's the case.

Reply:
Yeah - it falls back to os-client-config so it should be safe. We could potentially improve it to not throw the warning - however, the rest of the os_ modules all just require openstacksdk at this point and we really just missed this special snowflake when we made the other transition.